files/modrinth-exclude-include.json

```diff
@@ -32,6 +32,7 @@
     "continuity",
     "controlling",
     "craftpresence",
+    "crash-assistant",
```

Comment:
Unfortunately with Modrinth it needs to be part of the filename 

<img width="1206" height="1990" alt="Image" src="https://github.com/user-attachments/assets/f0680c9e-ac89-4997-9029-7e45bdfe97d2" />


```suggestion
    "CrashAssistant",
```